### DISCUSSION TITLE
How to output metrics of influx

### DISCUSSION THREAD

**Discussion by cosmind-rusu** [grafana/xk6-browser#909] [Q&A]
Hi, I've been searching and testing things but I can't find a way to do a test with k6browser and be able to export the metrics to an influxdb. There is an extension that already does that but it doesn't allow browser testing like k6browser. I have tried all this with docker, I hope help from the community thanks

**Comment by ankur22** [comment]
Hi @cosmind-rusu,

Thanks for the question. Could you please ask this question in the [community forum](https://community.k6.io/c/k6-browser/14)? If we find an issue on the back off investigating this, then we will open a new issue here, linking back to the forum post.

Could you also provide us with more detail on what you have tried, including the command line prompts and any relevant outputs (or screenshots)?

Thanks!
